src/components/book-content-mixin.cjsx

```diff
@@ -1,6 +1,9 @@
 _ = require 'underscore'
 S = require '../helpers/string'
 
+EXERCISE_LINK_SELECTOR = '.os-exercise .autogenerated-content'
```

Comment:
Why is this not `.os-exercise .os-embed`? (or `.os-exercise [data-type='problem'] > p > .os-embed`)? `.autogenerated-content` does not define the link as an exercise link